src/components/flex-container/flex-container.scss

```diff
@@ -30,7 +28,6 @@
 :host(limel-flex-container[align='stretch']) {
     align-items: stretch;
 }
-
```

Comment:
I've split this code formatting change into a separate commit from the functional change.